litmus-portal/graphql-server/manifests/cluster/3a_agents_rbac.yaml

```diff
@@ -280,4 +270,58 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: event-tracker-sa
-    namespace: #{AGENT-NAMESPACE}
\ No newline at end of file
+    namespace: #{AGENT-NAMESPACE}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: event-tracker-role
+rules:
+  # This role is used to access agent-config
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - list
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: event-tracker-role-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: event-tracker-role
+subjects:
+  - kind: ServiceAccount
+    name: event-tracker-sa
+    namespace: #{AGENT-NAMESPACE}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: subscriber-role
+  namespace: #{AGENT-NAMESPACE}
+  labels:
+    name: subscriber-role
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get","create","delete","update"]
```

Comment:
subscriber probably doesn't need to create and delete permissions get and update should be enough I believe. Are we deleting the configmap when remove agent action is called?